Core/src/TrackFitting/MbfSmoother.cpp

```diff
@@ -33,44 +33,54 @@ void MbfSmoother::visitMeasurement(const InternalTrackState& ts,
                                    BoundVector& small_lambda_hat) const {
   assert(ts.measurement.has_value());
 
-  const auto& measurement = ts.measurement.value();
+  const InternalTrackState::Measurement& measurement = ts.measurement.value();
+  const Acts::BoundMatrix& F = ts.jacobian;
 
   visit_measurement(measurement.calibratedSize, [&](auto N) -> void {
     constexpr std::size_t kMeasurementSize = decltype(N)::value;
 
+    using MeasurementMatrix =
+        Eigen::Matrix<ActsScalar, kMeasurementSize, eBoundSize>;
+    using CovarianceMatrix =
+        Eigen::Matrix<ActsScalar, kMeasurementSize, kMeasurementSize>;
+    using KalmanGainMatrix =
+        Eigen::Matrix<ActsScalar, eBoundSize, kMeasurementSize>;
+
     typename TrackStateTraits<kMeasurementSize, true>::Calibrated calibrated{
         measurement.calibrated};
     typename TrackStateTraits<kMeasurementSize, true>::CalibratedCovariance
         calibratedCovariance{measurement.calibratedCovariance};
 
     // Measurement matrix
-    const auto H = measurement.projector
-                       .template topLeftCorner<kMeasurementSize, eBoundSize>()
-                       .eval();
+    const MeasurementMatrix H =
+        measurement.projector
+            .template topLeftCorner<kMeasurementSize, eBoundSize>()
+            .eval();
 
     // Residual covariance
-    const auto S =
+    const CovarianceMatrix S =
         (H * ts.predictedCovariance * H.transpose() + calibratedCovariance)
             .eval();
     // TODO Sinv could be cached by the filter step
-    const auto S_inv = S.inverse().eval();
+    const CovarianceMatrix S_inv = S.inverse().eval();
 
     // Kalman gain
     // TODO K could be cached by the filter step
-    const auto K = (ts.predictedCovariance * H.transpose() * S_inv).eval();
+    const KalmanGainMatrix K =
+        (ts.predictedCovariance * H.transpose() * S_inv).eval();
 
-    const auto C_hat = (BoundMatrix::Identity() - K * H).eval();
-    const auto y = (calibrated - H * ts.predicted).eval();
+    const Acts::BoundMatrix C_hat =
+        (Acts::BoundMatrix::Identity() - K * H).eval();
+    const Eigen::Matrix<ActsScalar, kMeasurementSize, 1> y =
+        (calibrated - H * ts.predicted).eval();
 
-    const auto big_lambda_tilde =
+    const Acts::BoundMatrix big_lambda_tilde =
```

Comment:
Would you mind aligning these variable names with our convention, of you're already touching this code.

Reply:
switched to camelCase